apps/web/utils/logic/variableItem.ts

```diff
@@ -0,0 +1,110 @@
+// repo
+import {
+  VariableToSet,
+  VariableToSetType,
+  VariableValueLogicType,
+} from "@repo/data/useWalkthroughData";
+// local
+import { AnswerToCheckValueFn, AnswerTypes } from "../../stores/AnswerStore";
+import { isArray, isObject, isString } from "../typeChecking";
+// NOTE: this feels weird, but it makes sure this module is something we can spy on for tests
+import * as ThisModule from "./variableItem";
+
+export const getVariableItemValue = (
+  variableToSet: VariableToSet,
+  getAnswerToCheckValue: AnswerToCheckValueFn,
+): AnswerTypes => {
+  // check variableType
+  switch (variableToSet.variableType) {
```

Comment:
Given we only have one case in our switch could we just? 
```

if (variableToSet.variableType === VariableToSetType.Object) {
  return ThisModule.getVariableItemValueObject(variableToSet, getAnswerToCheckValue);
}

```

Reply:
Yeah, I had left this as is because there was a potential for another type, but at the moment there is not!